core/pom.xml

```diff
@@ -437,5 +437,36 @@
         </includes>
       </resource>
     </resources>
+    <pluginManagement>
+        <plugins>
+            <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
+            <plugin>
+                <groupId>org.eclipse.m2e</groupId>
+                <artifactId>lifecycle-mapping</artifactId>
+                <version>1.0.0</version>
+                <configuration>
+                    <lifecycleMappingMetadata>
+                        <pluginExecutions>
+                            <pluginExecution>
+                                <pluginExecutionFilter>
+                                    <groupId>org.codehaus.mojo</groupId>
+                                    <artifactId>
+                                        sql-maven-plugin
+                                    </artifactId>
+                                    <versionRange>[1.5,)</versionRange>
+                                    <goals>
+                                        <goal>execute</goal>
+                                    </goals>
+                                </pluginExecutionFilter>
+                                <action>
```

Comment:
Ok, then leave them in. They seemed semantically empty to me, but maybe they specify that the action to take is a "null action" / no-op